sdk/core/core-util/review/core-util.api.md

```diff
@@ -4,14 +4,21 @@
 
 ```ts
 
+import { AbortSignalLike } from '@azure/abort-controller';
+
 // @public
 export function computeSha256Hash(content: string, encoding: "base64" | "hex"): Promise<string>;
 
 // @public
 export function computeSha256Hmac(key: string, stringToSign: string, encoding: "base64" | "hex"): Promise<string>;
 
-// @public
-export function delay(timeInMs: number): Promise<void>;
+// Warning: (ae-forgotten-export) The symbol "DelayOptions" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export function delay<T>(timeInMs: number, options?: DelayOptions): Promise<T | void>;
+
+// @public (undocumented)
+export function delay<T>(timeInMs: number, value: T, options?: DelayOptions): Promise<T | void>;
```

Comment:
Let's not have a new `DelayOptions` interface, inlining should be sufficient.

Reply:
I'm tagging @joheredi in for the discussion on the interface

Reply:
@HarshaNalluru I thought it would be easier to read the docs to have the properties in an interface rather than inline. You only need to go and look at the interface definition if you are interested, otherwise just reading `DelayOptions` gives you all the info you need.

I don't have strong feelings about one or the other but I personally would go with the interface. Do you have any big concerns about having the interface?

Reply:
Just thinking prior art, no strong feelings either :)